public/scripts/frames.js

```diff
@@ -191,6 +192,11 @@ function fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, wor
     document.getElementById("logWorkTotalTime").innerHTML = logWorkTotalTime;
     document.getElementById("logWorkDone").value = workDone;
     document.getElementById("logWorkRT").innerHTML = estimatedTime - logWorkTotalTime;
+    var today = new Date();
+    var date = today.getDate() + "/" + (today.getMonth() + 1) + "/" + today.getFullYear();
+    var time = today.getHours() + ":" + today.getMinutes();
+    var commentDateTime = date + "|" + time + "|";
+    document.getElementById("logWorkComment").value = commentDateTime + assigneeTaskDescription.innerHTML + " : ";
```

Comment:
take the date from card description dont write again

Reply:
No change needed, since date in log work comment box should have the date when the work is being logged